common/buildings/wc_special_buildings.txt

```diff
@@ -60,12 +60,53 @@ gurubashi_arena_01 = {
 
 	type = special
 }
+########
+# Icecrown Citadel
+
+icecrown_citadel_01 = {
+	type_icon = "icon_structure_icecrown_citadel.dds"
+	
+	province_modifier = {
+		fort_level = 7
+	}
+
+	character_modifier = {
+		levy_reinforcement_rate = 0.1
+		levy_size = 0.1
+		martial = 2
+		monthly_dread = 0.1
+		vassal_limit = 5
+		vassal_levy_contribution_mult = 0.1
+		taiga_advantage = 4
+	}
+
+	type = special
+}
+
+########
+# Ironforge
+
+ironforge_01 = {
+	type_icon = "icon_structure_petra.dds"
+	
+	province_modifier = {
+		fort_level = 4
+		tax_mult = 0.6
+	}
+
+	character_modifier = {
+		development_growth_factor = 0.6
```

Comment:
Not going to lie. It's an insane bonus.

Reply:
i played a game of Ironforge, I am almost always in debt, and it balanced on the spreadsheet

Reply:
The point is it's an insane bonus in long play. There's no thing in the game that gives so much development growth. You need to combine a few factors to get such number.